packages/repository/src/decorators/metadata.ts

```diff
@@ -0,0 +1,42 @@
+import {InspectionOptions, MetadataInspector} from '@loopback/context';
+import {MODEL_KEY, MODEL_PROPERTIES_KEY} from '../';
+import {ModelDefinition, PropertyDefinition} from '../../index';
+
+export namespace ModelMetadata {
```

Comment:
I would prefer to create a `ModelMetadataHelper` class and make `getModelMetadata` a static method.

Since there is only one method, we can just simply export the `getModelMetadata` function.

IIRC, the naming convention for `namespace` should be lower case. `ModelMetadata.getModelMetadata()` will make us assume that `ModelMetadata` is a class and `getModelMetadata` is a static method.